sites/public/.env.template

```diff
@@ -1,7 +1,7 @@
 ## == IMPORTANT REMINDER: ANY EDITS HERE MUST BE UPDATED IN ALL ENVIRONMENTS (incl. CI) == ##
 BACKEND_API_BASE=http://localhost:3100
 BACKEND_PROXY_BASE=
-LISTINGS_QUERY=/listings/combined
+LISTINGS_QUERY=/listings
```

Comment:
Just noting that `ListingService` uses this as the root and automatically adds `/combined` when searching.